blog/draft/hoe-haven-jou-helpt.md

```diff
@@ -0,0 +1,117 @@
+---
+authors: [tom-ootes]
+tags: [haven, kubernetes, common-ground]
+draft: true
+---
+# Met Haven(+) word je écht digitaal autonoom, en kan je uiteindelijk zonder Amerikaanse cloudproviders
+
+Door alle perikelen die zich op het moment voltrekken in de Verenigde Staten staat het thema *Digitale Autonomie* ineens overal boven aan de agenda in overheids-it-land. Er wordt veel over gesproken, maar over hoe we precies onafhankelijk gaan worden lijkt nog geen consensus te bestaan.
+
+Voor complexe problemen als deze bestaat er nooit een silver bullet. Echter is er wel een project dat je helpt voorsorteren op de (nood)uitgang: de Kubernetes-standaard [Haven](https://haven.commonground.nl/). In dit artikel omschrijf ik welke rol ik voor Haven(+) zie als **oplossing** voor **Digitale Autonomie**. Haven is een standaard die bestaat uit [checks](https://haven.commonground.nl/techniek/checks) voor het correct en secure opzetten van Kubernetes clusters.
+
+<!-- truncate -->
+
+:::success[**TL;DR**]
+
+- Met de Haven-standaard heeft de Nederlandse overheid een praktisch instrument als stap naar Digitale Autonomie.
+- Door Kubernetes-clusters volgens de Haven-checks in te richten, wordt migratie naar Europese cloudproviders uiteindelijk eenvoudiger.
```

Comment:
"Europese cloudproviders" komt vaak terug, maar er wordt nergens een voorbeeld genoemd. Zijn er wel Europese cloudproviders? Zo ja, dan zou ik daar ook een verwijzing naar verwachten.

Reply:
Dit wil ik wel doen (ik heb er wat gevonden) maar mag dit wel op een overheidswebsite? Is dit dan geen reclame? Dat was een beetje mijn punt van zorg

Reply:
https://previder.nl/diensten/kubernetes-as-a-service > dit is bijvoorbeeld een Nederlandse aanbieder waarop je een Haven compliant Kubernetes cluster kan ozpetten